components/atom/tag/src/styles/index.scss

```
@@ -206,6 +209,7 @@ $base-class: '.sui-AtomTag';
       fill: $c;
 
       &#{$self}-actionable {
+        border: 1px solid;
```

```suggestion
        border: $bdw-s solid;
```

[SUI-Theme](https://github.com/SUI-Components/sui/blob/master/packages/sui-theme/src/settings/_box-style.scss#L25)

---

I missed this one! it was only for testing I will remove it!